test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -265,46 +268,54 @@ public void run(String[] args) {
             Point targetPoint = new Point(x + w / 2, y + h / 2);
 
             Robot robot = new Robot();
+            robot.setAutoWaitForIdle(true);
             robot.mouseMove(sourcePoint.x, sourcePoint.y);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
+            robot.delay(50);
+            robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             for (Point p = new Point(sourcePoint); !p.equals(targetPoint);
-                 p.translate(Util.sign(targetPoint.x - p.x),
+                p.translate(Util.sign(targetPoint.x - p.x),
                              Util.sign(targetPoint.y - p.y))) {
                 robot.mouseMove(p.x, p.y);
-                Thread.sleep(50);
+                robot.delay(50);
             }
 
             synchronized (Util.SYNC_LOCK) {
-                robot.mouseRelease(InputEvent.BUTTON1_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
                 Util.SYNC_LOCK.wait(Util.FRAME_ACTIVATION_TIMEOUT);
             }
 
-            if (!dragSourceListener.isDropFinished()) {
-                throw new RuntimeException("Drop not finished");
-            }
+            EventQueue.invokeAndWait(() -> {
+                if (!dragSourceListener.isDropFinished()) {
+                    throw new RuntimeException("Drop not finished");
+                }
+                success1 = dragSourceListener.getDropSuccess();
+                dragSourceListener.reset();
+            });
 
-            boolean success1 = dragSourceListener.getDropSuccess();
 
-            dragSourceListener.reset();
             robot.mouseMove(sourcePoint.x, sourcePoint.y);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
+            robot.delay(50);
+            robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             for (Point p = new Point(sourcePoint); !p.equals(targetPoint);
-                 p.translate(Util.sign(targetPoint.x - p.x),
+                p.translate(Util.sign(targetPoint.x - p.x),
                              Util.sign(targetPoint.y - p.y))) {
```

Comment:
```suggestion
                p.translate(Util.sign(targetPoint.x - p.x),
                            Util.sign(targetPoint.y - p.y))) {
```
Align the second parameter on the wrapped line to the first one on the first line.